scio-core/src/main/scala/com/spotify/scio/coders/CoderMaterializer.scala

```diff
@@ -77,7 +77,6 @@ object CoderMaterializer {
         )
       case Disjunction(typeName, idCoder, id, coders) =>
         WrappedBCoder.create(
-          // `.map(identity) is really needed to make Map serializable.
```

Comment:
Why remove this comment ? I remember you getting bitten by the `map` and adding this comment.

Reply:
uhh, oops (bad friday)... somehow i thought we ere not using `.map(identity)` anymore lol